### DISCUSSION TITLE
Simple brightness adaptions to survey images?

### DISCUSSION THREAD

**Discussion by GerritErdt** [Stellarium/stellarium#3313] [Ideas]
Hi all, 

today I noticed that the DSS2 Red survey, which I use most of the time to identify faint nebulas, shows much more faint detail when the atmosphere is active and the sky is not yet completely dark. That way, I can clearly see nebula structures that won't show without the additional brightness due to the atmosphere. 

To put it in technical terms: adding a bias to the survey data greatly improves the visibility of faint details. 

Is it possible to include such a custom adjustable bias into future versions for all the surveys? Or maybe even a simple, custom histogram stretch? Event the first, much easier option, would help to better see faint nebulas and thus enhance the planning capabilities stellarium offers. 

CS Gerrit

**Comment by gzotti** [comment]
continued in #3314
